src/main/java/com/atlassian/jira/cloud/jenkins/config/JiraCloudPluginConfig.java

```diff
@@ -25,15 +25,19 @@
     public static final String FIELD_NAME_AUTO_BUILDS = "autoBuilds";
     public static final String FIELD_NAME_AUTO_DEPLOYMENTS = "autoDeployments";
     public static final String FIELD_NAME_SITES = "sites";
+    public static final String FIELD_NAME_SITES2 = "sites2";
     public static final String FIELD_NAME_AUTO_BUILDS_REGEX = "autoBuildsRegex";
     public static final String FIELD_NAME_AUTO_DEPLOYMENTS_REGEX = "autoDeploymentsRegex";
 
     private static final Logger log = LoggerFactory.getLogger(JiraCloudPluginConfig.class);
 
     private static final String ATL_JSW_GLOBAL_CONFIGURATION_ID = "atl-jsw-global-configuration";
 
+    // TODO: remove the old "sites" field
     private List<JiraCloudSiteConfig> sites = new ArrayList<>();
 
+    private List<JiraCloudSiteConfig2> sites2 = new ArrayList<>();
```

Comment:
Stores the new type of Jira sites (with the webhook URL instead of the Client ID)